web/html/src/components/systems.js

```diff
@@ -64,6 +64,73 @@ function statusDisplay(system, isAdmin) {
   );
 }
 
+function SystemTabs(props) {
+    const sid = props.server.id;
+    const salt = props.server.entitlements.includes('salt_entitled');
+    const management = props.server.entitlements.includes('enterprise_entitled');
+    const virtualization = props.server.entitlements.includes('virtualization_host');
+    return (
+      <div className="spacewalk-content-nav">
```

Comment:
Do you think this list will grow? It feels a bit hard to read. What about a more declarative approach? 

```javascript
const allSystemTabs = [
  { 
    name: 'Configuration',   
    getClassname: ({active}) => active ?  'active' : 'configuration', 
    showCondition: ({server}) => server.features.ftr_config 
  },
  { 
    name: 'Provisioning',   
    getClassname: ({active}) => active ?  'active' : 'Provisioning', 
    showCondition: ({server}) => server.features.ftr_kickstart
  },
];

return (
   <div>
   {
      allSystemTabs.map((tab) => 
            tab.showCondition({server: props.server}) &&
                <li className={tab.getClassname({active: props.active})}>
                    <a href={tab.url}>{t(tab.name)}</a>
                </li>
   }
   </div>
)
```

Reply:
commit removed -> resolved